src/integrations/intercom/services/contact-getter.js

```diff
@@ -0,0 +1,42 @@
+const logger = require('../../../services/logger');
+
+const EMAIL_REGEX = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Looks very complex.
I know that the email format is complex, but shouldn't we simplify this and only check `@` presence (as you mentioned).
An `@` in the external id is something very unlikely to happen in my opinion, and thus, good enough.

WDYT?

Reply:
I think there is two ways to tackle the question:
- Considering that the ids we are talking about are unlikely to contain any `@` character and thus to do a simple check as you propose
- Considering that an email as a particular structure that goes beyond the simple `@` check (see the `EMAIL_REGEX`)

But after giving a thought to those two solutions, I think I'll go your way: maybe some users will try with dummy email structure that might not match the regex (ie. `a@a.a`)